bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -307,7 +307,10 @@ void ReplicaImp::tryToSendPrePrepareMsg(bool batchingLogic) {
     DebugStatistics::onSendPrePrepareMessage(pp->numberOfRequests(), requestsQueueOfPrimary.size());
   }
   primaryLastUsedSeqNum++;
-
+  {
+    concordlogger::MDC content(GL, "cid", "content: " + pp->getBatchCorrelationIdAsString());
+    LOG_INFO(GL, "");
```

Comment:
@yontyon, Won't this statement get logged for every `PrePrepareMsg` sent? If so, I suspect logging this statement at the `INFO` level will be incredibly noisy, contribute to clutter in the logs that makes them more cumbersome to interpret as a human, and accelerate the growth of log file size.

Reply:
Also @yontyon I am not sure if there is a need to output an INFO level message without any message body? Also, I am not sure if it's a standard to log without a message body.

Reply:
The purpose of this "empty" logline is to create a mapping between the batch correlation IDs and the batch sequence number.
I will change it to TRACE log info and add a simple message into the actual logline